wdl-engine/tests/inputs.rs

```diff
@@ -139,43 +145,55 @@ fn run_test(test: &Path, result: AnalysisResult, ntests: &AtomicUsize) -> Result
     }
 
     let document = result.document();
-    let result = match Inputs::parse(document, test.join("inputs.json")) {
-        Ok(Some((name, inputs))) => match inputs {
-            Inputs::Task(inputs) => {
-                match inputs
-                    .validate(
-                        document,
-                        document
-                            .task_by_name(&name)
-                            .expect("task should be present"),
-                        None,
-                    )
-                    .with_context(|| format!("failed to validate the inputs to task `{name}`"))
-                {
-                    Ok(()) => String::new(),
-                    Err(e) => format!("{e:?}"),
+
+    let json_path = test.join("inputs.json");
+    let yaml_path = test.join("inputs.yaml");
+
+    // Check if at least one of the input files exists
```

Comment:
Should we require both YAML and JSON be present? I suppose if we start adding other formats that might get annoying and difficult to maintain, but it seems preferable over having blind spots in our test coverage.

Reply:
I specifically requested adding both formats to the tests as 1) there aren't that many of these tests and 2) it ensures the same result regardless of input format.

I don't see us adding more formats for the foreseeable future, so I'd advocate to just keeping as-is in the PR.